package/entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+#########################################################################################################################################
+# DISCLAIMER																																																														#
+# Copied from https://github.com/moby/moby/blob/ed89041433a031cafc0a0f19cfe573c31688d377/hack/dind#L28-L37															#
+# Permission granted by Akihiro Suda <akihiro.suda.cz@hco.ntt.co.jp> (https://github.com/rancher/k3d/issues/493#issuecomment-827405962)	#
+# Moby License Apache 2.0: https://github.com/moby/moby/blob/ed89041433a031cafc0a0f19cfe573c31688d377/LICENSE														#
+#########################################################################################################################################
+if [ -f /sys/fs/cgroup/cgroup.controllers ]; then
+	# move the init process (PID 1) from the root group to the /init group,
+  # otherwise writing subtree_control fails with EBUSY.
+  mkdir -p /sys/fs/cgroup/init
+  echo 1 > /sys/fs/cgroup/init/cgroup.procs
+  sleep 3
```

Comment:
Why sleep 3?  On a very slow arm system will this fail?

Reply:
My bad, leftover from testing, where I failed to understand for a while why this just didn't want to work but worked on second try. Those were the issues with PID1 :roll_eyes: 
```suggestion
```